.github/workflows/maintainer-edit.yml

```diff
@@ -3,8 +3,14 @@ on:
   pull_request_target:
     types:
       - opened
+permissions:
+  contents: read
+
```

Comment:
I don't believe we need content read permissions for this workflow, and permissions for the job are defined below, so we should be able to omit this.

```suggestion
```